assets/js/scroll-to-top-button.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-env browser */
+
+// Get the button:
+var mybutton = document.getElementById("scrollToTopBtn");
+
+// When the user scrolls down 20px from the top of the document, show the button
+window.onscroll = function() {
+  scrollFunction()
+};
+
+topFunction();
```

Comment:
I think this makes the browser call this method everytime this file is loaded. Are you sure you want this?

Reply:
Well, I didn't add that in at first...but CodeFactor is complaining that `topFunction()` is not called.

Happy for suggestions as I'm not too sure what to do :frowning_face: